dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryDirectory.java

```diff
@@ -152,7 +152,9 @@ private String judgeCategory(URL url) {
         return "";
     }
 
-    private void refreshOverrideAndInvoker(List<URL> urls) {
+    // Issue: https://github.com/apache/dubbo/issues/6763.

```

Comment:
remove this line, and we can refer these changes in git blame

Reply:
done.